sharding-core/src/main/java/in/cleartax/dropwizard/sharding/hibernate/MultiTenantSessionFactoryHealthCheck.java

```diff
@@ -85,7 +85,7 @@ public Void run() {
                         sessionFactory.getCurrentSession().createNativeQuery(validationQuery).list();
                         return null;
                     }
-                }.start(false, new DefaultUnitOfWorkImpl());
+                }.start(false, new DefaultUnitOfWorkImpl(), "healthCheck");
```

Comment:
This is not backwards compatible. GST would also need changes. Better to have an overload.